acstech/doppler-frontend
# Javascript restructure and dynamic event list update

Changes:
Condensed the websocket, modal, and animation javascript files into one file all of which is used in local scope (the user cannot access these functions via the browser console).
Updates event list based on websocket messages (only non-initial events cause a new events notification to pop up). 
Fixed "disappearing" points.
Optimized javascript in the condensed file and map file in order to improve user experience.
Added dynamic modal creation and destruction functions.
Updated javascript function documentation.
Fixes #99 
To Test:
Remove all but one event from nav1.
Then log into nav1 (make sure that the frontend API is running) on the frontend and start up the backend API and then the testSend file. Next go to the filter list on the frontend and see the notification that a new event was added.
